cypress/plugins/index.js

```diff
@@ -15,7 +20,78 @@
 /**
  * @type {Cypress.PluginConfig}
  */
-module.exports = (_on, _config) => {
+module.exports = (on, config) => {
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
+  config.env = {
+    ...config.env,
+    ...process.env,
+  };
+
+  on('task', {
+    async resetData() {
```

Comment:
If we start to add more tasks, I will pull this out. For now, I'm leaving it here as it's simpler as it has access to `config.env`.